Tests/TestTriggerChallenges.cs

```diff
@@ -0,0 +1,703 @@
+using System;
+using System.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using PrivacyIDEA_Client;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+using WireMock.Server;
+
+namespace Tests
+{
+    [TestClass]
+    public class TestTriggerChallenges
+    {
+#pragma warning disable CS8618 // [TestInitialize] acts as constructor
+        WireMockServer server;
+        PrivacyIDEA privacyIDEA;
+#pragma warning restore CS8618 // [TestInitialize] acts as constructor
+
+        [TestInitialize]
+        public void Setup()
+        {
+            server = WireMockServer.Start();
+            privacyIDEA = new PrivacyIDEA(server.Urls[0], "test", false);
+        }
+
+        [TestCleanup]
+        public void Cleanup()
+        {
+            server.Stop();
+        }
+
+        [TestMethod]
+        public async Task TriggerChallenges()
+        {
+// Test no service account
```

Comment:
Can you extract each of the comments to a function with that name

Reply:
[8d9d7b4](https://github.com/privacyidea/csharp-client/pull/10/commits/8d9d7b460d2fcb694b8bc2006585cc5541a0b129)